### PR TITLE
Update autocruadmin version to 1.1.28

### PR DESCRIPTION
**Pull request by msmner**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/847

Linked pull requests from another repos (if any):

**Summary of the changes made**:
1. Updated Autocrudadmin version to 1.1.29
2. Fix build errors in administration

### CODE REVIEW

File: Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs [hunk 1]
```diff
@@ -536,11 +542,13 @@ protected override async Task BeforeEntitySaveOnCreateAsync(Problem entity, Admi
...
-
-        if (!originalEntity.ProblemGroup.Contest.IsOnlineExam)
+        if (newEntity.ProblemGroup != null)
         {
-            newEntity.ProblemGroup.OrderBy = newEntity.OrderBy;
+            newEntity.ProblemGroup.Type = actionContext.GetProblemGroupType().GetValidTypeOrNull();
+            if (originalEntity.ProblemGroup != null && !originalEntity.ProblemGroup.Contest.IsOnlineExam)
+            {
+                newEntity.ProblemGroup.OrderBy = newEntity.OrderBy;
+            }
```
Comment (gogo4ds) [inline Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs:551]: Why this is part of this PR?

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs (hunk 1 above)
Comment (msmner) [reply Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs]: Removed as discussed.

**Review by shsimeonova — APPROVED** [review]

**Review by shsimeonova — APPROVED** [review]

File: Servers/Administration/OJS.Servers.Administration.Infrastructure/OJS.Servers.Administration.Infrastructure.csproj [hunk 2]
```diff
@@ -11,7 +11,7 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="AutoCrudAdmin" Version="1.1.26" />
+      <PackageReference Include="AutoCrudAdmin" Version="1.1.28" />
```
Comment (gogo4ds) [inline Servers/Administration/OJS.Servers.Administration.Infrastructure/OJS.Servers.Administration.Infrastructure.csproj:14]: We have newer version - 1.1.29

**Review by gogo4ds — CHANGES_REQUESTED** [review]

**Comment by msmner** [comment]
Updating to 1.1.29 was done in another PR, so I am closing this one.

**Pull request closed without merging**